app/Repositories/PostRepository.php

```diff
@@ -200,7 +200,10 @@ public function reviews(Post $post, $status, $comment = null, $admin = null)
         $post->status = $status;
         $post->save();
 
-        // If the Post has a school ID, update School's aggregate impact for the Action.
+        // Update the "counter cache" on the Post Campaign:
+        $post->campaign->refreshCounts();
```

Comment:
I'm trying to think of if there is anyway a post could get reviewed without hitting this function - but I guess vast majority would come through here so we probably would never be too far off?